SonarRunner.Shim/SonarRunner.Wrapper.cs

```diff
@@ -67,25 +67,34 @@ public ProjectInfoAnalysisResult Execute(AnalysisConfig config, IEnumerable<stri
 
```

Comment:
I see what you mean, but the validation not only looks for a sonar-project.properties file in the SonarRunnerWorkingDirectory, but also in any of the folder containing a .csproj file (the "module" folders). The list of modules is loaded in `Execute()`, and not `ExecuteJavaRunner()`, which is why I placed the validation logic there. It should be doable however with a bigger refactoring.